src/pretix/control/templates/pretixcontrol/event/plugins.html

```diff
@@ -10,20 +10,41 @@ <h1>{% trans "Available plugins" %}</h1>
             software functionality, connect your event to third-party services, or apply other forms of customizations.
         {% endblocktrans %}
     </p>
+    {% if "success" in request.GET %}
+        <div class="alert alert-success">
+            {% trans "Your changes have been saved." %}
+        </div>
+    {% endif %}
+    <p>
+        <div class="row">
+            <div class="col-lg-10">
+                <input type="search" id="plugin_search_input" class="form-control" placeholder="{% trans "Search" %}" autofocus>
+            </div>
+            <div class="col-lg-2">
+                <div class="checkbox">
+                    <label><input type="checkbox" id="plugin_only_active"> {% trans "Show only active plugins" %}</label>
```

Comment:
I don't like how this looks, the vertical positions feel quite misaligned. Don't have a perfect idea for a fix though:

![image](https://github.com/user-attachments/assets/03c22fa9-fb57-47c5-9761-c12d08e93e72)

Get's even weirder on smaller screens:

![image](https://github.com/user-attachments/assets/ddcac3fd-4ac6-47c7-82c3-9f980c977058)

Reply:
I changed it to a button-radio-group (see screenshot in comment)